app/controllers/password_resets_controller.rb

```diff
@@ -37,6 +38,12 @@ def user_params
     params.require(:user).permit(:password, :password_confirmation)
   end
 
+  def find_token
+    @user = User.find_by(email: params[:email])
```

Comment:
You can save this step by searching for the user based on the `hashed_token` instead of the email.